benchmark/dss/20200812/src/main/java/MyServiceResponse.java

```diff
@@ -0,0 +1,13 @@
+import io.github.ztkmkoo.dss.core.actor.rest.entity.DssRestServiceResponse;
+
+public class MyServiceResponse implements DssRestServiceResponse {
```

Comment:
DssRestServiceResponse is serializable.
You should add serialVersionUID

Reply:
ref link: https://woowabros.github.io/experience/2017/10/17/java-serialize2.html